detection_rules/etc/version.lock.json

```diff
@@ -9558,32 +9572,32 @@
       }
     },
     "rule_name": "Disabling User Account Control via Registry Modification",
-    "sha256": "a99de43d815eee8fe71df7ef937c7ab3fafe8bd0d195d2ad3d7d8b0fb9c1cf90",
+    "sha256": "e3673f075041b1dcb529fa9445d9e7f022636aa583a56d6f325190852b415b07",
     "type": "eql",
-    "version": 102
+    "version": 103
   },
   "d331bbe2-6db4-4941-80a5-8270db72eb61": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Clearing Windows Event Logs",
-        "sha256": "dec588e701c38e67a10b0a6521c4c022892f6671b2feca4f727a6d17df173077",
+        "sha256": "f60152637e6804eaa8df1e4b003a7b6f42b4ae55bc5214071a76d06d100b4f92",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.